gobblin-cluster/src/test/java/org/apache/gobblin/cluster/GobblinHelixJobLauncherTest.java

```diff
@@ -175,7 +171,7 @@ public void run() {
     this.thread.start();
   }
 
-  private Properties generateJobProperties(Config baseConfig, String jobNameSuffix, String jobIdSuffix) {
+  static public Properties generateJobProperties(Config baseConfig, String jobNameSuffix, String jobIdSuffix) {
```

Comment:
You made this `static public` for testing right? If so, nit: you can make it package private and use also use `visiblefortesting` annotation to make that clearer.